channels/generic/http.py

```diff
@@ -81,7 +81,7 @@ async def http_request(self, message):
                 await self.handle(b"".join(self.body))
             finally:
                 await self.disconnect()
-                raise StopConsumer()
+            raise StopConsumer()
```

Comment:
what a difference an indentation level makes 😅

Reply:
💝 — This may be the root of one I've been meaning to sit down with for an age. 

I shall take a proper look later one.

Reply:
> So there may be failing HTTP consumers in production apps where the exceptions aren’t being logged.

Yes, that was the reported issue.

Reply:
Little dance 🕺